package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "sprint-mission2",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

README.md에 실행 방법을 잘 작성해주셨는데, package.json의 scripts를 활용하실 수도 있어요.

```suggestion
  "scripts": {
    "start": "node --env-file=.env src/main.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
```